src/btree/bt_sync.c

```diff
@@ -137,6 +137,57 @@ err:	WT_TRET(__wt_page_release(session, next, flags));
 }
 
 /*
+ * __sync_checkpoint_progress --
+ * 	Output a checkpoint progress message.
+ */
+static void
+ __sync_checkpoint_progress(WT_SESSION_IMPL *session)
+{
+	struct timespec cur_time;
+	WT_CONNECTION_IMPL *conn;
+	uint64_t time_diff;
+
+	conn = S2C(session);
+	time_diff = 0;
+	__wt_epoch(session, &cur_time);
+
+	/* Time since the full database checkpoint started */
+	time_diff = WT_TIMEDIFF_SEC(cur_time, conn->ckpt_verb_start_time);
+
+	if ((time_diff / 20) > conn->ckpt_progress_msg_count) {
+		__wt_verbose(session, WT_VERB_CHECKPOINT_PROGRESS, "Checkpoint"
```

Comment:
@agorrod I am not sure if there are strong guidelines on how to structure such multiline statements. Your point of view will be appreciated